File: Python/EstruturaDeDecisao/exercicio_04.py
"""
Faça um Programa que verifique se uma letra digitada é vogal ou consoante.
"""

letra = str(input('Digite a letra: '))

if letra.isalpha() and len(letra) == 1:
    if letra.lower() in 'aeiou':
        print('A letra é: vogal')
    else:
        print('A letra é: consoante')
else:
    print('Entrada inválida.')

File: Java/EstruturaDeDecisao/Exercicio04.java
// Faça um Programa que verifique se uma letra digitada é vogal ou consoante.
package EstruturaDeDecisao;
import java.util.Scanner;

public class Exercicio04 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite uma letra: ");
        char letra = scanner.next().charAt(0);

        letra = Character.toLowerCase(letra);

        if (letra == 'a' || letra == 'e' || letra == 'i' || letra == 'o' || letra == 'u') {
            System.out.println("A letra é uma vogal.");
        } else if (letra >= 'a' && letra <= 'z') {
            System.out.println("A letra é uma consoante.");
        } else {
            System.out.println("Caractere inválido.");
        }

        scanner.close();
    }
}
